File: atmospheric_calculations.cs
using System;

class Program
{
    static void Main()
    {
        Console.WriteLine("\nSıcaklık Ve Yoğunluk Verilirse (2)");
        Console.WriteLine("Sıcaklık Ve Basınç Verilirse (3)");
        Console.WriteLine("Yoğunluk Ve Basınç Verilirse (4)");
        Console.WriteLine("\nStratosfer için Hesaplar... [11-25km] (5)\n");

        Console.Write("\nSEÇİMİ YAPINIZ ! \n");
        int a = Convert.ToInt32(Console.ReadLine());

        if (a == 1)
        {
            Console.Write('\nYüksekliği Gir\n');
            double h = Convert.ToDouble(Console.ReadLine());
            double bb = Math.Pow((1 - h / 44308), 5.259);
            double b = bb * 101325;
            double yy = Math.Pow((1 - h / 44308), 4.259);
            double y = yy * 1.225;
            double tt = (bb * 101325) / (287 * 1.225 * yy);
            double ttd = tt - 273;
            Console.WriteLine($"\nBasınç Oranı p/p0 : {bb:F3} \n");
            Console.WriteLine($"Yoğunluk Oranı ro/ro0 : {yy:F3} \n");
            Console.WriteLine($"Sıcaklık : {tt:F3} Kelvin\n");
            Console.WriteLine($"Sıcaklık : {ttd:F3} Derece\n");
            Console.WriteLine($"Yoğunluk ({h} İrtifada) : {y:F3} kg/m^3\n");
            Console.WriteLine($"Basınç ({h} İrtifada): {b:F3} Pascal\n");
        }
        else if (a == 2)
        {
            Console.Write("Sıcaklığı (x metre) Girin : \n");
            double tt = Convert.ToDouble(Console.ReadLine());
            Console.Write("Yoğunluğu (x metre) Girin : \n");
            double y = Convert.ToDouble(Console.ReadLine());
            double p = 287 * tt * y;
            double bb = Math.Pow((p / 101325), (1.0 / 5.329));
            double h = (bb - 1) * (-44308);
            Console.WriteLine($"Yükseklik : {h:F3} m\n");
            Console.WriteLine($"Basınç : {p:F3} Pascal");
        }
        else if (a == 3)
        {
            Console.Write("Sıcaklığı (x metre) Girin : \n");
            double tt = Convert.ToDouble(Console.ReadLine());
            Console.Write("Basınç (x metre) Girin : \n");
            double p = Convert.ToDouble(Console.ReadLine());
            double ro = p / (287 * tt);
            double yy = Math.Pow((ro / 1.225), (1.0 / 4.259));
            double h = (yy - 1) * (-44308);
            Console.WriteLine($"Yükseklik : {h:F3} m\n");
            Console.WriteLine($"Yoğunluk : {ro:F3} kg/m^3\n");
        }
        else if (a == 4)
        {
            Console.Write("Yoğunluğu (x metre) Girin : \n");
            double y = Convert.ToDouble(Console.ReadLine());
            Console.Write("Basınç (x metre) Girin : \n");
            double p = Convert.ToDouble(Console.ReadLine());
            double tt = p / (y * 287);

            double bb = Math.Pow((p / 101325), (1.0 / 5.329));
            double h = (bb - 1) * (-44308);

            double yy = Math.Pow((y / 1.225), (1.0 / 4.259));
            h = (yy - 1) * (-44308);

            Console.WriteLine($"Yükseklik : {h:F3} m\n");
            Console.WriteLine($"Sıcaklık : {tt:F3} K\n");
        }
        else if (a == 5)
        {
            Console.Write("Yüksekliği Girin : \n");
            double h = Convert.ToDouble(Console.ReadLine());
            double x = (-1.5776 * Math.Pow(10, -4)) * (h - 11000);
            double ssag = Math.Exp(x);
            double y = ssag * 0.363;
            double b = ssag * 22591.749;
            double tt = b / (y * 287);
            Console.WriteLine($"Yoğunluk : {y:F3} kg/m^3\n");
            Console.WriteLine($"Basınç : {b:F3} Pascal\n");
            Console.WriteLine($"Sıcaklık : {tt:F3} K\n");
        }
        else
        {
            Console.WriteLine("Yanlış Bir Rakam Girdiniz");
        }
    }
}

File: atmospheric_calculations.m

fprintf('\nYükseklik Verilirse (1)');
fprintf('\nSıcaklık Ve Yoğunluk Verilirse (2)');
fprintf('\nSıcaklık Ve Basınç Verilirse (3)');
fprintf('\nYoğunluk Ve Basınç Verilirse (4)\n');

fprintf('\nStratosfer için Hesaplar... [11-25km] (5)\n');

a = input('\nSEÇİMİ YAPINIZ ! \n');

if a == 1;
    
    h = input('\nYüksekliği Gir\n');
    bb = (1 - h / 44308)^5.259;
    b = bb*101325;
    yy = (1 - h / 44308)^4.259;
    y = yy*1.225;
    tt = (bb*101325 )/(287*1.225*yy);
    ttd = tt-273;
    fprintf("\nBasınç Oranı p/p0 : %.3f \n",bb);
    fprintf("Yoğunluk Oranı ro/ro0 : %.3f \n",yy);
    fprintf("Sıcaklık : %.3f Kelvin\n",tt);
    fprintf("Sıcaklık : %.3f Derece\n",ttd);
    fprintf("Yoğunluk (%d İrtifada) : %.3f kg/m^3\n",h,y);
    fprintf("Basınç (%d İrtifada): %.3f Pascal\n",h,b);
    
elseif a == 2;
    tt = input('Sıcaklığı (x metre) Girin : \n');
    y = input('Yoğunluğu (x metre) Girin : \n');
    p = 287*tt*y;
    bb = (p / 101325)^(1 / 5.329);
    h = (bb-1)*(-44308);
    fprintf("Yükseklik : %.3f m\n",h);
    fprintf("Basınç : %.3f Pascal",p);
    
elseif a == 3;
    tt = input('Sıcaklığı (x metre) Girin : \n');
    p = input('Basınç (x metre) Girin : \n');
    ro = p/(287*tt);
    yy = (ro / 1.225)^(1 / 4.259);
    h = (yy-1)*(-44308);
    fprintf("Yükseklik : %.3f m\n",h);
    fprintf("Yoğunluk : %.3f kg/m^3\n",ro);
    
elseif a == 4;
    y = input('Yoğunluğu (x metre) Girin : \n');
    p = input('Basınç (x metre) Girin : \n');
    tt = p /( y*287);
    
    bb = (p / 101325)^(1 / 5.329);
    h = (bb-1)*(-44308);
    
    yy = (y / 1.225)^(1 / 4.259);
    h = (yy-1)*(-44308);
    
    fprintf("Yükseklik : %.3f m\n",h);
    fprintf("Sıcaklık : %.3f K\n",tt);
elseif a == 5;
    h = input('Yüksekliği Girin : \n');
    x = (-1.5776*10^(-4))*(h-11000);
    ssag = exp(x);
    y = ssag*0.363;
    b = ssag*22591.749;
    tt = b/(y*287);
    fprintf("Yoğunluk : %.3f kg/m^3\n",y);
    fprintf("Basınç : %.3f Pascal\n",b);
    fprintf("Sıcaklık : %.3f K\n",tt);
    
    

    
else 
    fprintf("Yanlış Bir Rakam Giridniz");
end
